src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs

```diff
@@ -842,9 +849,9 @@ private void UpdateContentLayoutSlots(Rect frame)
 		/// </summary>
 		internal void ClearMeasuredSize() => _measuredContentSize = null;
 
-		public override UICollectionViewLayoutAttributes PreferredLayoutAttributesFittingAttributes(UICollectionViewLayoutAttributes layoutAttributes)
+		public override UICollectionViewLayoutAttributes PreferredLayoutAttributesFittingAttributes(UICollectionViewLayoutAttributes nativeLayoutAttributes)
 		{
-			if (!(((object)layoutAttributes) is UICollectionViewLayoutAttributes))
+			if (((object)nativeLayoutAttributes) is not _LayoutAttributes)
```

Comment:
nit: how about? avoiding doing another casting on the line below
```suggestion
			if (((object)nativeLayoutAttributes) is not _LayoutAttributes layoutAttributes)
```